staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -6618,6 +6618,10 @@ type NodeSystemInfo struct {
 	Architecture string `json:"architecture" protobuf:"bytes,10,opt,name=architecture"`
 	// Swap Info reported by the node.
 	Swap *NodeSwapStatus `json:"swap,omitempty" protobuf:"bytes,11,opt,name=swap"`
+	// Whether the node is running in a user namespace.
+	// +featureGate=KubeletInUserNamespace
```

Comment:
clear this field in `nodeStatusStrategy#PrepareForUpdate` if the feature is disabled?

```go
	if !utilfeature.DefaultFeatureGate.Enabled(features.KubeletInUserNamespace) && oldNode.Status.NodeInfo.RunningInUserNamespace == nil {
		newNode.Status.NodeInfo.RunningInUserNamespace = nil
	}
```